sickbeard/tv.py

```diff
@@ -708,6 +708,7 @@ def load_episodes_from_db(self):
                 cur_ep.load_from_indexer(tvapi=t, cached_season=cached_seasons[cur_season])
                 scanned_eps[cur_season][cur_episode] = True
             except EpisodeDeletedException:
+                # Catch if episode got deleted while load_from_indexer
```

Comment:
we already catch `EpisodeDeletedException` here that's why I removed the code in 774.
It was making we call `load_from_indexer` twice every single episode